library/Rules/Image.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+class Image extends AbstractRule
+{
+    public function validate($input)
+    {
+    	if (is_file($input) || filter_var($input, FILTER_VALIDATE_URL)) {
+     	   return is_array(getimagesize($input));
```

Comment:
I don't think a good idea to require GD installed as dependency just to execute a validation rule, instead you can use [finfo](http://php.net/finfo) which is already in use by [Mimetype](https://github.com/Respect/Validation/blob/master/library/Rules/Mimetype.php) rule.